build.gradle

```diff
@@ -2,16 +2,16 @@ buildscript {
     apply from: 'versions.gradle'
 
     repositories {
-        maven { url "https://plugins.gradle.org/m2/" }
-        jcenter()
         google()
+        mavenCentral()
+        maven { url "https://jitpack.io" }
+        gradlePluginPortal()
```

Comment:
why are we adding this?

Reply:
`google()` is the same as maven `{ url "https://maven.google.com" }`

`gradlePluginPortal()` is the same as `maven { url "https://plugins.gradle.org/m2/" }`